code/WorkInProgress/simroom.dm

```diff
@@ -252,28 +252,44 @@
 		boutput(M, SPAN_NOTICE("<B>You cannot possibly fit into that!</B>"))
 		return
 
+	if(src.emagged && iscarbon(M))
+		boutput(M, SPAN_ALERT("<b>You feel a terrible pain in your head, and everything goes black...<b/>"))
+		M.setStatusMin("unconscious", 5 SECONDS) //lets give them *some* chance eh?
```

Comment:
the comment suggests that we're giving people a chance but I don't see how the related code accomplishes this

Reply:
the old code would knock you out for i think 10 seconds?

Reply:
need to check further, but basically allow them *some* chance to get up before getting fully mauled by the lion they spawned right next to